### PR TITLE
Fixed forms for Flabébé and Furfrou

### PR DESCRIPTION
**Pull request by Fabio1988**
@ReuschelCGN @jfberry does this makes sense?

It's also needed to change the basic english name, instead "red" would be translated to "Rotblütler". Because the forms in english were "diamond" or "red" in this repo.... But it's "Diamond Trim" and "Red Flower".

[english Flabebe](https://bulbapedia.bulbagarden.net/wiki/Flab%C3%A9b%C3%A9_(Pok%C3%A9mon))
[german Flabebe](https://www.pokewiki.de/Flab%C3%A9b%C3%A9)

[english Furfrou](https://bulbapedia.bulbagarden.net/wiki/Furfrou_(Pok%C3%A9mon))
[german Furfrou](https://www.pokewiki.de/Coiffwaff)

### CODE REVIEW

**Comment by TurtIeSocks** [comment]
@Fabio1988 I think your local version of pogo-translations may have an older version of PDG? There seems to be a lot of missing protos in this PR.

**Comment by Fabio1988** [comment]
I did a npm install, let me try again :)

**Comment by Fabio1988** [comment]
> ├─┬ pogo-data-generator@1.3.9
> │ └─┬ pogo-protos@2.55.8 (github:Furtif/pogo-protos#441ef92c543f5104ba6795818219fa6cbed72f1b)

**Comment by Fabio1988** [comment]
I think it's because the repo refer to 2.55.1 and not 2.55.8 protos

> pogo-translations@0.2.0 /home/fabio/git/pogo-translations
> └─┬ pogo-data-generator@1.3.9
>   ├─┬ node-fetch@2.6.7
>   │ └─┬ whatwg-url@5.0.0
>   │   ├── tr46@0.0.3
>   │   └── webidl-conversions@3.0.1
>   └─┬ pogo-protos@2.55.1 (github:Furtif/pogo-protos#d95f57ab27da3fda15cdf64a9209ee80e4624866)
>

**Review by TurtIeSocks — APPROVED** [review]
